js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,17 @@
+const animalSort = animalsArray => {
+  if (!Array.isArray(animalsArray)) {
+    throw new TypeError(`Expected array as a function parameter, got ${typeof animalsArray}`);
+  }
+  animalsArray.sort((obj1, obj2) => {
```

Comment:
Instead of generic objects, we are comparing 2 animals. This would have been a little better:
```suggestion
  animalsArray.sort((animal1, animal2) => {
```